packages/ui/src/components/va-tree-view/VaTreeView.demo.vue

```diff
@@ -1,59 +1,340 @@
-        EXPAND ALL
-      </va-button>
-      <va-button @click="$refs.treeView.collapse()">
-        COLLAPSE ALL
-      </va-button>
-
-      <va-tree-view ref="treeView">
-        <va-tree-category
-          label="Category text"
-          is-open
-        >
-          <va-tree-node>
-            Node text
-          </va-tree-node>
-          <va-tree-category
-            label="Category text"
-            is-open
-          >
-            <va-tree-node>
-              Node text
-            </va-tree-node>
-          </va-tree-category>
-        </va-tree-category>
-      </va-tree-view>
+    <VbCard title="Default">
+      <va-tree-view :nodes="nodes" />
+    </VbCard>
+    <VbCard title="Stateless">
+      <va-tree-view :nodes="nodes" :stateful="false" />
+    </VbCard>
+    <VbCard title="Selectable">
+      <p>Default</p>
+      <va-tree-view :nodes="nodes" selectable />
+      <p>Selection strategy</p>
+      <va-radio
+        v-for="(option, index) in selectionTypeOptions"
+        :key="index"
+        v-model="selectionType"
+        :option="option"
+      />
+      <va-tree-view
+        :nodes="nodes"
+        :selection-type="selectionType"
+        selectable
+      />
+    </VbCard>
+    <VbCard title="Colored checkboxes">
+      <va-color-palette
+        v-model="selectedColor"
+        :palette="colorsPalette"
+      />
+      <va-tree-view
+        :nodes="nodes"
+        selectable
+        :color="selectedColor"
+      />
+    </VbCard>
+    <VbCard title="Selection model">
+      <p>Selection strategy</p>
+      <va-radio
+        v-for="(option, index) in selectionTypeOptions"
+        :key="index"
+        v-model="selectionType"
+        :option="option"
+      />
+      <p>
+        Single selection model for two tree views:
+        <br />
+        {{ JSON.stringify(selectedNodes) }}
+      </p>
+      <va-tree-view
+        v-model:checked="selectedNodes"
+        :nodes="nodes"
+        :selection-type="selectionType"
+        selectable
+      />
+      <va-tree-view
+        v-model:checked="selectedNodes"
+        :selection-type="selectionType"
+        :nodes="nodes"
+        selectable
+      />
+    </VbCard>
+    <VbCard title="valueBy">
+      <p>{{ JSON.stringify(selectedNodesValueBy) }}</p>
+      <br />
+      <va-tree-view
+        v-model:checked="selectedNodesValueBy"
+        :nodes="nodes"
+        :value-by="({id, label}) => `${id}_${label}`"
+        selectable
+      />
+    </VbCard>
+    <VbCard title="textBy">
+      <va-tree-view
+        :nodes="nodesTextBy"
+        text-by="title"
+      />
     </VbCard>
-    <VbCard title="Color">
-      <va-tree-view
-        color="info"
-      >
-        <va-tree-category
-          label="Category text"
-          is-open
-        >
-          <va-tree-node>
-            Node text
-          </va-tree-node>
-          <va-tree-node>
-            Node text
-          </va-tree-node>
-        </va-tree-category>
+    <VbCard title="trackBy">
```

Comment:
<img width="229" alt="image" src="https://user-images.githubusercontent.com/5394573/191760685-cb78e6b3-5138-4508-a7b8-3b401f9c204c.png">

This demo doesn't seem to test functionality